samples/sensor/fxls8974/sample.yaml

```diff
@@ -0,0 +1,26 @@
+common:
+  harness: console
+  harness_config:
+    type: multi_line
+    ordered: true
+    regex:
+      - ".*Booting.*"
+      - "^AX=\\W*([0-9]|1[0-1])\\..*AY=\\W*([0-9]|1[0-1])\\..*AZ=\\W*([0-9]|1[0-1])\\..*CROSS=\\W*(10[0-5]|9[0-9]).*$"
+      - "^AX=\\W*([0-9]|1[0-1])\\..*AY=\\W*([0-9]|1[0-1])\\..*AZ=\\W*([0-9]|1[0-1])\\..*CROSS=\\W*(10[0-5]|9[0-9]).*$"
+      - "^AX=\\W*([0-9]|1[0-1])\\..*AY=\\W*([0-9]|1[0-1])\\..*AZ=\\W*([0-9]|1[0-1])\\..*CROSS=\\W*(10[0-5]|9[0-9]).*$"
+      - "^AX=\\W*([0-9]|1[0-1])\\..*AY=\\W*([0-9]|1[0-1])\\..*AZ=\\W*([0-9]|1[0-1])\\..*CROSS=\\W*(10[0-5]|9[0-9]).*$"
+      - "^AX=\\W*([0-9]|1[0-1])\\..*AY=\\W*([0-9]|1[0-1])\\..*AZ=\\W*([0-9]|1[0-1])\\..*CROSS=\\W*(10[0-5]|9[0-9]).*$"
+  tags: sensors
+  depends_on:
+    - i2c
+    - gpio
+sample:
+  name: FXOS8974 Accelerometer
+tests:
+  sample.sensor.fxls8974.accel:
+    filter: dt_compat_enabled("nxp,fxls8974")
+  sample.sensor.fxls8974.accel.trigger:
+    extra_configs:
+      - CONFIG_FXLS8974_TRIGGER_OWN_THREAD=y
+      - CONFIG_FXLS8974_DRDY_INT1=y
+    skip: true # remove after a board with connected interrupt is added
```

Comment:
is this need, if the filter is there?

Reply:
Yes, some boards do not have interrupt pins connected to the accelerometer.

Reply:
then please use platform_exclude, with this setting all boards will be skipped.